src/style.css

```
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-small,
+.search-big {
+  position: relative;
+  display: block;
+}
+
+.search-big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-big input,
```

use class selectors instead of tag selectors
```suggestion
.search-big .input,
```

---

is not fixed